packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -69,6 +71,11 @@ export class AuthenticatorComponent
       formFields,
     } = this;
 
+    configureComponent({
+      packageName: '@aws-amplify/ui-angular',
+      version: VERSION,
+    });
+
```

Comment:
Verified the change locally: 
![image](https://user-images.githubusercontent.com/43682783/207717763-70720811-0256-4304-ae41-d2d86429c5f5.png)